typescript/sdk/src/deploy/HyperlaneAppChecker.ts

```diff
@@ -126,18 +126,24 @@ export abstract class HyperlaneAppChecker<
     return filterOwnableContracts(contracts);
   }
 
-  // TODO: Require owner in config if ownables is non-empty
-  async checkOwnership(chain: ChainName, owner: types.Address): Promise<void> {
+  async checkOwnership(
+    chain: ChainName,
+    owner: types.Address,
+    ownableOverrides?: Record<string, types.Address>,
+  ): Promise<void> {
     const ownableContracts = await this.ownables(chain);
     for (const [name, contract] of Object.entries(ownableContracts)) {
+      const expectedOwner = ownableOverrides
+        ? ownableOverrides[name] ?? owner
+        : owner;
```

Comment:
Tip, optional chaining works for object field access too: `const expectedOwner = ownableOverrides?.[name] ?? owner`

Reply:
nice